encryption-decryption/src/com/nbugaenco/encryptdecrypt/Main.java

```diff
@@ -0,0 +1,119 @@
+package com.nbugaenco.encryptdecrypt;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class Main {
+
+    private static final Map<Parameter, String> params = new HashMap<>() {{
+        put(Parameter.MODE, "enc");
+        put(Parameter.KEY, "0");
+        put(Parameter.DATA, "");
+        put(Parameter.ALG, "shift");
+        put(Parameter.IN, null);
+        put(Parameter.OUT, null);
+    }};
+
+    public static void main(String[] args) {
```

Comment:
I see that overall logic works, but try to move away from ``Main.java`` this "Mode-related logic", and try to add some comments with documentation where you think it worth adding

Reply:
In this case it's returning just array of bytes (byte[]), so I need to make a String from it. Or make a new constructor...